### PR TITLE
CLOUD-225: first draft

### PR DESCRIPTION
**Pull request by svenwltr**
> https://jira.rebuy.de/browse/CLOUD-225

There is a lot room for improvements, but I think we should do that later.

Missing features:
- tests
- removal of old volumes

### CODE REVIEW

**Comment by svenwltr** [comment]
@rebuy-de/it-platform Please review.

File: main.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+const (
+	PAUSE = 1 * time.Minute
```
Comment (ghost) [inline main.go]: do you really need the 1*

Comment (svenwltr) [inline main.go]: no

**Comment by ghost** [comment]
:+1:

**Comment by florian-zeidler** [comment]
👍 LGTM

**Pull request merged**